### PR TITLE
Landing page responsive

### PR DESCRIPTION
**Pull request by xenia11**
## Rationale

Create the landing page responsive for all devices.

## Advice for Reviewers & Testing Notes

You can test the responsiveness of the page by changing the size of the screen.

## Screenshots:

- Add a screenshot or two of your changes, where applicable.

## Task Name

[react] UI Design - responsive landing page #37

## Linting Checklist
- [X ] No commented code
- [X ] Code Formatted nicely (Prettier)
- [X ] PR your own code before you assign reviewers

### CODE REVIEW

**Comment by xenia11** [comment]
Hi guys. I made some changes on the landing page by making it responsive. Let me know If I need to change something.

**Comment by deBubz** [comment]
hey @xenia11 can you re-open the pr into `dev`

**Comment by xenia11** [comment]
I'll resolve conflicts this evening.

File: web/src/pages/landing/Landing.jsx [hunk 1]
```diff
@@ -34,23 +31,25 @@ function Landing() {
...
                     </Grid>
                 </Grid>
-                <Grid container spacing={4} style={{ paddingTop: "4em" }}>
-                    {cardData.map((item, index) => (
-                        <Card item={item} index={index} />
-                    ))}
+                <Grid container className="classes.second">
+                    <Grid container spacing={4} style={{ paddingTop: "4em" }}>
+                        {cardData.map((item, index) => (
+                            <Card item={item} index={index} key={index} />
```
Comment (kimchirice) [inline web/src/pages/landing/Landing.jsx:50]: [MD]

one minor, considering to add a new key, such as *ID* in the cardData. then apply the *id* as key in each <Card>

reference [indexs as a key is anti pattern](https://reactpatterns.js.org/docs/indexes-as-a-key-is-an-anti-pattern/)

**Comment by deBubz** [comment]
nice work, ive solved the conflicts and added a hover change on the nav links

**Pull request merged**